src/main/java/net/snowflake/client/jdbc/DefaultSFConnectionHandler.java

```diff
@@ -110,6 +118,100 @@ protected void initialize(SnowflakeConnectString conStr, String appID, String ap
     }
   }
 
+  private void setClientConfig() throws SnowflakeSQLLoggedException {
+    Map<SFSessionProperty, Object> connectionPropertiesMap = sfSession.getConnectionPropertiesMap();
+    String clientConfigFilePath =
+        (String) connectionPropertiesMap.getOrDefault(SFSessionProperty.CLIENT_CONFIG_FILE, null);
+
+    SFClientConfig sfClientConfig;
+    try {
+      sfClientConfig = SFClientConfigParser.loadSFClientConfig(clientConfigFilePath);
+    } catch (IOException e) {
+      throw new SnowflakeSQLLoggedException(sfSession, ErrorCode.INTERNAL_ERROR, e.getMessage());
+    }
+    sfSession.setSfClientConfig(sfClientConfig);
+  }
+
+  /**
+   * This method instantiates a JDK14Logger. This will be used if the java.util.logging.config.file
+   * properties file is missing. The method performs the following actions: 1. Check if the
+   * CLIENT_CONFIG_FILE is present. If it is, the method loads the logLevel and logPath from the
+   * client config. 2. Check if the Tracing parameter is present in the URL or connection
+   * properties. If it is, the method will overwrite the logLevel obtained from step 1. 3.
+   * Instantiate java.util.logging with the specified logLevel and logPath. 4. If both the logLevel
+   * and logPath are null, this method doesn't do anything.
+   */
+  private void initLogger() throws SnowflakeSQLLoggedException {
+    if (logger instanceof JDK14Logger
+        && systemGetProperty("java.util.logging.config.file") == null) {
+      Map<SFSessionProperty, Object> connectionPropertiesMap =
+          sfSession.getConnectionPropertiesMap();
+      String tracingLevelFromConnectionProp =
+          (String) connectionPropertiesMap.getOrDefault(SFSessionProperty.TRACING, null);
+
+      Level logLevel = null;
+      String logPattern = "%t/snowflake_jdbc%u.log"; // default pattern.
```

Comment:
if %t for time and %u for user then can we switch it? like %u/snowflake_jdbc_%t?

Reply:
@sfc-gh-igarish We could. But we currently use `%t/snowflake_jdbc%u.log` this format so I kept it as-is for backward compatibility. 
ref: https://github.com/snowflakedb/snowflake-jdbc/blob/master/src/main/java/net/snowflake/client/log/JDK14Logger.java#L288